fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/query/AggregateValueTest.java

```diff
@@ -100,6 +103,19 @@ void testSumOnlyNulls() {
         accumulateAndAssert(new NumericAggregationValue.Sum(PhysicalOperator.SUM_D, ofScalar(1D)), doublesOnlyNull, (Object)null);
     }
 
+    @Test
+    void testBitMap() {
+        accumulateAndAssertByteArray(new NumericAggregationValue.BitMap(PhysicalOperator.BITMAP_LL, ofScalar(1)), bitsetForBitMap(new Object[]{0L, 1L, 2L, 0L}), Arrays.asList(0L, 1L, 2L)); // 111
+        accumulateAndAssertByteArray(new NumericAggregationValue.BitMap(PhysicalOperator.BITMAP_LL, ofScalar(1)), bitsetForBitMap(new Object[]{0L, 1L, 2L, 0L, 64L, 65L, 66L}), Arrays.asList(0L, 1L, 2L, 64L, 65L, 66L)); // 111
+        accumulateAndAssertByteArray(new NumericAggregationValue.BitMap(PhysicalOperator.BITMAP_LL, ofScalar(1)), bitsetForBitMap(new Object[]{0L, 1L, 2L, 0L, 64L, 65L, 66L, 10000L, 10001L, 10100L}), Arrays.asList(0L, 1L, 2L, 64L, 65L, 66L, 10000L, 10001L, 10100L)); // 111
```

Comment:
Does this test the endianness as well? could you please make sure this is covered as well? It looks to me, in the assertion logic, you're iterating over the bytes and returning a list of set bits as long, that seems to glance over endianness's requirement.

Bottom line is, we must make sure the order of returned bytes of the streaming aggregate version of `bitmap` is aligned with what the indexed version of it returns.

Reply:
I think the `collectOnBits` includes the endianness, if the endianness is incorrect, it wouldn't match the expected result.
```
for (int i = 0; i < bitmap.length; i++) {
            if (bitmap[i] != 0) {
                for (int j = 0; j < 8; j++) {
                    if ((bitmap[i] & (1 << j)) != 0) {
                        result.add(i * 8L + j);
                    }
                }
            }
        }
```